WalletWasabi.Fluent/ViewModels/AddWallet/Create/ConfirmRecoveryWordsViewModel.cs

```diff
@@ -133,8 +133,10 @@ private void EnableAvailableWords(bool enable)
 
 	private async Task OnNextAsync()
 	{
-		var dialogCaption = "This is needed to open and to recover your wallet. Store it safely, it cannot be changed.";
-		var password = await Navigate().To().CreatePasswordDialog("Add Password", dialogCaption, enableEmpty: true).GetResultAsync();
+		var dialogCaption = "It's a recovery words extension for more security.\n" +
+			"You will need it to open and to recover your wallet.\n" +
```

Comment:
```suggestion
			"You will need it to spend your Bitcoin or recover your wallet.\n" +
```

Reply:
In Wasabi it is first needed to open the wallet, otherwise users cannot spend their bitcoin.

Reply:
The user is automatically logged in after wallet creation now, so they don't first need it to open the wallet, right?  https://github.com/zkSNACKs/WalletWasabi/pull/11390